api/odilo.py

```diff
@@ -200,26 +185,27 @@ def extract_date(cls, data, field_name):
         return d
 
     @classmethod
-    def raise_exception_on_error(cls, data, custom_error_to_exception={}):
+    def raise_exception_on_error(cls, data, default_exception_class=None, ignore_exception_code=None):
```

Comment:
Since your'e changing the API, ignore_exception_code should be a list of exceptions to ignore, not a single item.

ignore_exception_code is unusual enough that I would value a description of it in the docstring. Something like "These error codes do not represent exceptional conditions and will be ignored rather than transformed into exceptions."

Reply:
ok, changed!